src/core/term.ml

```diff
@@ -9,10 +9,38 @@
 
 open Timed
 
-module Tags = Parsing.Syntax.Tags
-
 (** {3 Term (and symbol) representation} *)
 
+(** Symbol modifiers and attributes. *)
+module Tags = struct
```

Comment:
What's the point of this module? I would prefer to be able to access its types directly.